src/components/list/themes/shared/item/list-item.bootstrap.scss

```diff
@@ -11,11 +11,18 @@ $theme: $bootstrap;
     border-bottom: none;
 }
 
-
 [part='title'] {
     @include type-style('body-1');
 }
 
 [part='subtitle'] {
     @include type-style('body-2');
 }
+
+[name='start']::slotted(*) {
```

Comment:
@desig9stein You don't need these margins here. You already have them in the list-item.common